src/HotChocolate/Neo4J/src/Data/Language/Visitor/CypherVisitor.VisitationMap.cs

```diff
@@ -0,0 +1,231 @@
+using System.Linq;
+
+namespace HotChocolate.Data.Neo4J.Language
+{
+    public partial class CypherVisitor
```

Comment:
At some point we can align this with the core and visit from the out side in. 
Then you can just instantiate a visitor and override what you need